src/api/app/views/webui2/webui/kiwi/images/_build_info.html.haml

```diff
@@ -0,0 +1,39 @@
+%ul.nav.nav-tabs.pt-2.px-3.flex-nowrap.collapsible
+  %li.nav-item
+    %a.nav-link.active.result-select-link#result-select-link{ href: '#' } Build Results
+
+.result-display
+.card-body
+  .btn.btn-sm.btn-outline-primary.build-refresh{ onclick: 'update_build_result()', title: 'Refresh Build Results' }
+    Refresh
+    %i.fas.fa-sm.fa-sync-alt{ id: 'result_spinner' }
+%hr.m-0
+#result-display
+
+:ruby
```

Comment:
same here!

Reply:
Linter shouts at me even more after removing that ruby tag :/